src/Simplex/Chat/Archive.hs

```diff
@@ -94,53 +94,60 @@ copyDirectoryFiles fromDir toDir = do
 
 deleteStorage :: ChatMonad m => m ()
 deleteStorage = do
-  StorageFiles {chatDb, agentDb, filesPath} <- storageFiles
-  removeFile chatDb
-  removeFile agentDb
+  fs@StorageFiles {filesPath} <- storageFiles
+  liftIO $ closeSQLiteStore `withStores` fs
+  removeFile `withDBs` fs
   mapM_ removePathForcibly filesPath
   tmpPath <- readTVarIO =<< asks tempDirectory
   mapM_ removePathForcibly tmpPath
 
 data StorageFiles = StorageFiles
   { chatDb :: FilePath,
-    chatEncrypted :: TVar Bool,
+    chatStore :: SQLiteStore,
```

Comment:
if it's being refactored anyway, why keep chatDb, agentDb separately in StorageFiles

Reply:
withDBs could still exist and take the from stores